pytest_watch/watcher.py

```diff
@@ -88,7 +88,8 @@ def run(self, summary=None):
                     msg = ('Changes detected, rerunning: {}'
                            .format(highlight(command)))
             print(STYLE_NORMAL + msg + Fore.RESET + Style.NORMAL)
-        exit_code = subprocess.call(['py.test'] + self.args, shell=True)
+        exit_code = subprocess.call(['py.test'] + self.args,
+                                    shell=subprocess.mswindows)
```

Comment:
Wow, never knew about `subprocess.mswindows`. There's so many ways to test what OS you're on. This one definitely seems the most explicit when doing a Windows-only check.